openlibrary/plugins/openlibrary/js/darkMode.js

```diff
@@ -0,0 +1,17 @@
+export function initDarkMode() {
+    const body = document.querySelector('body');
+    const icon = document.getElementById('darkModeIcon')
+    const cookie = document.cookie
+    if (!cookie.includes('dm=True')) {
+        const dm = 'dm'
+        const darkModeValue ='True';
+        const cookieStr = `${dm}=${encodeURIComponent(darkModeValue)}`;
+        document.cookie = cookieStr;
```

Comment:
```suggestion
        document.cookie = 'dm=True;';
```
I don't think that `encodeURIComponent` is needed here.  This function is more useful for modifying URLs.

Reply:
thanks Jim! I wasn't aware that it was much simpler to change the cookie